src/locales/strings/enUS.json

```diff
@@ -668,7 +668,7 @@
   "transaction_details_fee_warning": "High Network Fees",
   "transaction_details_swap": "Swap Funds",
   "transaction_details_swap_network_fee": "Swap Network Fee",
-  "transaction_details_swap_order_cancel": "Swap Order Canceled",
```

Comment:
Other languages?

Reply:
You need to replace this string in other languages that are untranslated

Reply:
What do you mean by "replace the string"?